packages/shorebird_cli/lib/src/commands/patch/patch_android_archive_command.dart

```diff
@@ -0,0 +1,394 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:archive/archive_io.dart';
+import 'package:crypto/crypto.dart';
+import 'package:http/http.dart' as http;
+import 'package:mason_logger/mason_logger.dart';
+import 'package:path/path.dart' as p;
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/config/config.dart';
+import 'package:shorebird_cli/src/formatters/file_size_formatter.dart';
+import 'package:shorebird_cli/src/shorebird_artifact_mixin.dart';
+import 'package:shorebird_cli/src/shorebird_build_mixin.dart';
+import 'package:shorebird_cli/src/shorebird_code_push_client_mixin.dart';
+import 'package:shorebird_cli/src/shorebird_config_mixin.dart';
+import 'package:shorebird_cli/src/shorebird_create_app_mixin.dart';
+import 'package:shorebird_cli/src/shorebird_environment.dart';
+import 'package:shorebird_cli/src/shorebird_validation_mixin.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+/// {@template patch_android_archive_command}
+/// `shorebird patch android_archive`
```

Comment:
I know it's a significant change but what are your thoughts on adjusting `shorebird release android` and `shorebird patch android` to accept a flag like `module`? that way developers using add-to-app would do:

```sh
shorebird release android --module
shorebird patch android --module
```

This is similar to how a Flutter app identifies itself as a "module" in the pubspec:
https://github.com/flutter/samples/blob/aff3b0979e14aee958219ab3bbf936b38df0dba4/add_to_app/fullscreen/flutter_module/pubspec.yaml#L32

Reply:
based on our convo sounds like we'll rename to `shorebird release aar` 👍

Reply:
Done.